js/components/dataset/quality.vue

```diff
@@ -1,57 +1,68 @@
 <template>
 <div>
 <box :title="title" icon="thumbs-up" boxclass="box-solid" v-if="quality">
-    <doughnut :score="quality.score + 1" width="200px" height="200px"></doughnut>
```

Comment:
We can't have a full doughnut with the max score (9) today. I think we should somehow standardize the score (between 0 and 10?).

Reply:
WDYT in the last commit?